GooglePlayInstant/Editor/PluginUpdateWindow.cs

```diff
@@ -333,6 +333,22 @@ public static void ShowWindow()
             window.minSize = new Vector2(WindowMinWidth, WindowMinHeight);
         }
 
+        // Returns a new LatestReleaseResponse. This is primarily here to set fields and avoid Unity warning CS0649.
+        private static LatestReleaseResponse CreateLatestReleaseResponse(string version, string downloadUrl)
```

Comment:
Alternatively, you could suppress the warnings by declaring those classes inside a pragma warning block:
```
#pragma warning disable CS0649
// code with warnings
#pragma warning restore CS0649
```

Reply:
Hmm. I considered that originally, but then decided against it. Given how verbose my solution turned out to be I think you're right. I'll switch to that.